docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -404,3 +404,233 @@ Example output:
 ```text
 SpecVersion=2.0
 ```
+
+## TPM009.001 CREATEPRIMARY Function (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that CREATEPRIMARY function works as expected.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+1. Install `tpm2-tools` using:
```

Comment:
Maybe if in each of the test case `tpm2_tools` is needed it would be better to add this to the common section?

Reply:
Not every test case needs `tpm2_tools`